packages/insomnia/src/ui/components/settings/account.tsx

```diff
@@ -1,278 +1,186 @@
-import { autoBindMethodsForReact } from 'class-autobind-decorator';
-import React, { Fragment, PureComponent } from 'react';
-import { connect } from 'react-redux';
+import React, { FC, Fragment, useCallback, useState } from 'react';
+import { useSelector } from 'react-redux';
 
 import * as session from '../../../account/session';
-import { AUTOBIND_CFG } from '../../../common/constants';
-import { RootState } from '../../redux/modules';
 import { selectSettings } from '../../redux/selectors';
 import { Link } from '../base/link';
 import { PromptButton } from '../base/prompt-button';
 import { HelpTooltip } from '../help-tooltip';
 import { hideAllModals, showModal } from '../modals/index';
 import { LoginModal } from '../modals/login-modal';
 
-type ReduxProps = ReturnType<typeof mapStateToProps>;
+export const Account: FC = () => {
+  const settings = useSelector(selectSettings);
```

Comment:
suggestion: it's best if we can avoid leaking the full settings object.  before this PR, we accomplished this in this file like this:

```ts
  disablePaidFeatureAds: selectSettings(state).disablePaidFeatureAds,
```

we could still do something similar, and at least reduce the scope by keeping the full settings object from being exposed to the rest of this file.

```suggestion
  const { disablePaidFeatureAds } = useSelector(selectSettings);
```